pkgs/tools/compression/dejsonlz4/default.nix

```diff
@@ -10,7 +10,7 @@ stdenv.mkDerivation rec {
     };
 
     buildPhase = ''
-      gcc -Wall -o dejsonlz4 src/dejsonlz4.c src/lz4.c
+      ${stdenv.cc.targetPrefix}cc -o dejsonlz4 src/dejsonlz4.c src/lz4.c
```

Comment:
I'm curious what the difference to `$CC` is in this case.

Reply:
Personal preference. $CC would work as well but I like to be more explicit about cross compilation prefixes.